components/concept/storage/src/main/java/mozilla/components/concept/storage/HistoryStorage.kt

```diff
@@ -165,6 +172,18 @@ data class TopFrecentSiteInfo(
     val title: String?
 )
 
+/**
+ * Frecency threshold options for fetching top frecent sites. Requests a page that was visited
+ * with a frecency score greater or equal to the [value].
+ */
+enum class FrecencyThresholdOption(val value: Long) {
+    /** Returns all visited pages. */
+    NONE(0),
+
+    /** Skip visited pages that were only visited once. */
+    SKIP_ONE_TIME_PAGES(101)
```

Comment:
I don't think you need the actual frecency scores here? These are already encapsulated in the A-S counterpart.

Reply:
Removed the value.